shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -115,6 +115,6 @@ FLUTTER_DARWIN_EXPORT
  * }
  * ```
  */
-@property(readwrite, nonatomic, nullable) NSColor* backgroundColor;
+@property(readwrite, nonatomic, nullable, strong) NSColor* backgroundColor;
```

Comment:
Use `copy` here rather than `strong`. NSColor is immutable but there's no guarantee someone hasn't subclassed it. (IIRC NSColor is actually a tagged pointer these days so yeah, this is probably inconsequential, but really copy is the right semantics to express).